packages/ui-components/src/components/UIActionCallout/UIActionCallout.tsx

```diff
@@ -69,6 +72,7 @@ export class UIActionCallout extends React.Component<ActionCalloutProps> {
         this.icon = props.icon;
         this.onClick = props.onClick;
         this.anchorClicked = false;
+        this.isError = props.isError || false;
```

Comment:
Can you use the operator (`??`) instead of a logical or (`||`), as it is a safer operator

Reply:
Thanks @eouin, I have added the requested change.